test/XRP/serializer.test.ts

```diff
@@ -164,6 +174,127 @@ function makeDepositPreauth(
   return transaction
 }
 
+/**
+ * Make a Transaction representing an AccountSet operation.
+ *
+ * @param clearFlagValue - The ClearFlag value to use for the AccountSet.
+ * @param domainValue - The Domain value to use for the AccountSet.
+ * @param emailHashValue - The EmailHash value to use for the AccountSet.
+ * @param messageKeyValue - The MesssageKey value to use for AccountSet.
+ * @param setFlagValue - The SetFlag value to use for the AccountSet.
+ * @param transferRateValue - The TransferRate value to use for the Accountset.
+ * @param tickSizeValue - The TickSize value to use for the AccountSet.
+ * @param fee - The amount of XRP to use as a fee, in drops.
+ * @param lastLedgerSequenceNumber - The last ledger sequence the transaction will be valid in.
+ * @param sequenceNumber - The sequence number for the sending account.
+ * @param senderAddress - The address of the sending account.
+ * @param publicKey - The public key of the sending account, encoded as a hexadecimal string.
+ * @returns A new `Transaction` object comprised of the provided properties.
+ */
+function makeAccountSetTransaction(
+  clearFlagValue: number | undefined,
+  domainValue: string | undefined,
+  emailHashValue: Uint8Array | undefined,
+  messageKeyValue: Uint8Array | undefined,
+  setFlagValue: number | undefined,
+  transferRateValue: number | undefined,
+  tickSizeValue: number | undefined,
+  fee: string,
+  lastLedgerSequenceNumber: number,
+  sequenceNumber: number,
+  senderAddress: string | undefined,
+  publicKey: string,
+): Transaction {
+  const accountSet = makeAccountSet(
+    clearFlagValue,
+    domainValue,
+    emailHashValue,
+    messageKeyValue,
+    setFlagValue,
+    transferRateValue,
+    tickSizeValue,
+  )
+  const transaction = makeBaseTransaction(
+    fee,
+    lastLedgerSequenceNumber,
+    sequenceNumber,
+    senderAddress,
+    publicKey,
+  )
+  transaction.setAccountSet(accountSet)
+
+  return transaction
+}
+
+/**
+ * Make an AccountSet protocol buffer fro the given inputs.
```

Comment:
nit: `for`

Reply:
Done. I used `from` instead of `for`. LMK if that distinction bugs you.